src/OpenTelemetry.Contrib.Instrumentation.Runtime/RuntimeMetrics.cs

```diff
@@ -81,6 +83,11 @@ public RuntimeMetrics(RuntimeMetricsOptions options)
                 this.meter.CreateObservableGauge($"{metricPrefix}memory.usage", () => (double)(Environment.WorkingSet / 1_000_000), "MB", "Working Set");
             }
 
+            if (options.IsCpuEnabled)
+            {
+                this.meter.CreateObservableGauge($"{metricPrefix}cpu.usage", this.CalculateCpuUsage, "CPU usage");
```

Comment:
Perhaps this should be a [process metric](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/process-metrics.md#process) rather than a runtime metric? This measurement has no information that is specific to the .NET runtime, it is solely extracting information that the OS tracks for every process. 
https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/207#discussion_r818272869